extensions/default/src/getCustomizationModule.tsx

```diff
@@ -25,5 +25,39 @@ export default function getCustomizationModule() {
         ],
       },
     },
+    {
+      name: 'default',
+      value: [
+        {
+          id: 'ohif.overlayItem',
+          uiType: 'uiType',
+          content: function(props) {
+            if (this.condition && !this.condition(props)) return null;
+
+            const { instance } = props;
+            const value =
+              instance && this.attribute
+                ? instance[this.attribute]
+                : this.contentF && typeof this.contentF === 'function'
+                ? this.contentF(props)
+                : null;
+            if (!value) return null;
+
+            return (
+              <span
+                className="overlay-item flex flex-row"
+                style={{ color: this.color || undefined }}
+                title={this.title || ''}
+              >
+                {this.label && (
+                  <span className="mr-1 shrink-0">{this.label}</span>
+                )}
+                <span className="font-light">{value}</span>
+              </span>
+            );
+          },
+        },
+      ],
+    },
```

Comment:
can you add a comment above that what should be added where for people to use this customization in their mode?